### DISCUSSION TITLE
Alignments not working

### DISCUSSION THREAD

**Discussion by syed-qure** [facebook/lexical#3736] [Q&A] [answered]
I'm trying to use left/right/center align commands in my editor. But they are not working.

How I'm triggering commands
```javascript
editor.dispatchCommand(FORMAT_ELEMENT_COMMAND, 'center')
```

Please help

**Comment by PunkFleet** [comment]
I have not tried to build the toolbar manually, but this is the official toolbar template:
https://github.com/facebook/lexical/blob/main/packages/lexical-playground/src/plugins/ToolbarPlugin/index.tsx

**Reply by syed-qure** [reply]
Thanks for your response. 

I have the same code wrt to align btns but it does not work for me. What I think I'm missing is configuring `ParagraphNode` since it is responsible for the alignment via `text-align` property

**Reply by PunkFleet** [reply]
I use playground's sample code, which, although not comprehensive enough, meets my needs

**Comment by syed-qure** [comment, marked as answer]
The fix was to import `FORMAT_ELEMENT_COMMAND` from proper package

```javascript
import { FORMAT_ELEMENT_COMMAND } from 'lexical'
```
